hack/generator/pkg/astmodel/type.go

```diff
@@ -31,6 +31,19 @@ type Type interface {
 	Equals(t Type) bool
 }
 
+// Types is the set of all types being generated
+type Types map[TypeName]TypeDefinition
+
+// Add adds a type to the set, with sanity check
```

Comment:
The term "sanity check" has some baggage, best to avoid ...

```suggestion
// Add adds a type to the set, with safety check that we haven't already added that type
```

Reply:
good point

Reply:
updated all instances